lib/web_ui/test/text_editing_test.dart

```diff
@@ -1577,7 +1577,9 @@ void testMain() {
           <String, dynamic>{
             'text': 'something',
             'selectionBase': 9,
-            'selectionExtent': 9
+            'selectionExtent': 9,
+            'composingBase': -1,
+            'composingExtent': -1
```

Comment:
Let's add a test (or more) that sends a few composition events and makes sure they are handled correctly.